config/prod/synapse-login-scipoolprod-params.yaml

```diff
@@ -26,13 +26,13 @@ sceptre_user_data:
     #  273957 Sage Bionetworks
     # 3409010 scipoolprod-internal
     # 3409011 scipoolprod-external
-    # 3469335 Schwannomatosis Open Research Collaborative Service Catalog Users
+    # 3451917 SWNTS (Schwannomatosis Open Research Collaborative)
     - Name: TeamToRoleArnMap
       Value: >-
         '[
           {"teamId":"3407239","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"273957","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"3409010","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogEndusers"},
           {"teamId":"3409011","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"},
-          {"teamId":"3469335","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"}
+          {"teamId":"3451917","roleArn":"arn:aws:iam::237179673806:role/ServiceCatalogExternalEndusers"}
```

Comment:
Is this intentional?  I believe the `ServiceCatalogExternalEndusers` role only has access to a limited set of products.  It does not have access to all the products that `ServiceCatalogEndusers` role does.

Reply:
Yes, it's intentional.  @allaway says, in IT-2718:

> we have a couple participants outside the US who don’t have enough local compute to explore some of the data files. We are hoping to make an rstudio instance or two available that they could use (something like a c5.xlarge or 2xlarge, nothing crazy).

So giving them access just to EC2 is sufficient.